src/lib/api/index.js

```diff
@@ -1,8 +1,11 @@
 import React, { Component } from 'react'
-import { getContent, setContent, setLocale } from '../reducers'
+import { setContent, unsetContent, setLocale } from '../reducers'
 
 import { locations, getJsApiName, getReactApiName } from 'lib/api/helpers'
 
+// The React API need unique IDs, so we use an iterator here
+let IDiteror = 0
```

Comment:
It is not an iterator (nor an iteror :D).

I would just call this `let id = 0`

Reply:
I would prefer `idToIncrement` to be more explicit, but agree about the iterator naming here. Thanks!